apps/web/components/getting-started/steps-views/ConnectedVideoStep.tsx

```diff
@@ -36,9 +41,13 @@ const ConnectedVideoStep = (props: ConnectedAppStepProps) => {
                     <AppConnectionItem
                       type={item.type}
                       title={item.name}
+                      isDefault={item.slug === metadata?.defaultConferencingApp?.appSlug}
                       description={item.description}
                       logo={item.logo}
                       installed={item.userCredentialIds.length > 0}
+                      defaultInstall={
+                        (item.isOAuth || item.slug === "google-meet") && !hasAnyInstalledVideoApps
```

Comment:
I don't like checking the type of an app here. We should really check for the feature of the app.

I believe the apps that we want to support here to be set as default are those that have `isOAuth=true` i.e. they have been setup using a personal connection. But Google Meet doesn't have `isOAuth` set to true, because it is setup automatically with Google Calendar. Daily Video too doesn't have isOAuth set to true.
<img width="573" alt="Screenshot 2024-07-09 at 8 46 36 AM" src="https://github.com/calcom/cal.com/assets/1780212/e68bad68-256c-4b30-916c-64e0e3f70bff">

I think the right check here could be `metadata.appData.location.linkType==='dynamic'`. `dynamic` means the link is per booking and not a static link i.e. common for all bookings.

Reply:
Yes this would make more sense, based on earlier conversation I assumed we were only adding it for oauth based apps.